TESTING.md

```diff
@@ -53,22 +55,21 @@ To spin up a development environment for the *jenkins-datadog-plugin* repository
 
 5. Access your Jenkins instance http://localhost:8080
 6. Enter the administrator password in the Getting Started form.
-7. On the next page, click on the "Select plugins to be installed" unless you want to installing suggested plugins. 
-8. Selected desired plugins depending on your needs. You can always add plugins later.
-  - You can skip user creation and stick to the admin user.
-9. Create a user so that you don't have to use the admin credential again (optional)
+7. On the next page, click on the "Select plugins to be installed" unless you want to install all suggested plugins. 
+8. Select desired plugins depending on your needs. You can always add plugins later.
+9. Create a user so that you don't have to use the admin credential again (optional).
 10. Continue until the end of the setup process and log back in.
 11. Go to http://localhost:8080/configure to configure the "Datadog Plugin", set your `API Key`.
-  - Click on the "Test Key" to make sure your key is valid
-  - You can set your machine `hostname`
-  - `.*, owner:$1, release_env:$2, optional:Tag3`
-  - You can add `node` optional tag by clicking on the node checkbox
+  - Click on the "Test Key" to make sure your key is valid.
+  - You can set your machine `hostname`.
+  - `.*, owner:$1, release_env:$2, optional:Tag3`.
```

Comment:
Is this for the fields textbox? Can we clarify that at the beginning of this line?

Reply:
Oups Good catch!